README.md

```diff
@@ -1,4 +1,17 @@
-# ifk-articles
+<h1 align="center">ingenjorsarbete-for-klimatet.github.io</h1>
```

Comment:
We could just go with 

```suggestion
<h1 align="center">ingenjorsarbeteforklimatet.se</h1>
```

as the repo name is forced to be ingenjorsarbete-for-klimatet.github.io.

Reply:
Sure. I wanted to display the lint and build status in the readme file,  but I guess I messed it up somehow.